Makefile

```diff
@@ -253,6 +257,21 @@ test:
 test-unstructured-api-unit:
 	scripts/test-unstructured-api-unit.sh
 
+.PHONY: test-no-extras
+# TODO(newelh) Add json test when fixed
+test-no-extras:
+	PYTHONPATH=. CI=$(CI) pytest \
+		test_${PACKAGE_NAME}/partition/test_text.py \
+		test_${PACKAGE_NAME}/partition/test_email.py \
+		test_${PACKAGE_NAME}/partition/test_html_partition.py \
+		test_${PACKAGE_NAME}/partition/test_xml_partition.py 
+
+.PHONY: test-extra
+test-extra:
+	PYTHONPATH=. CI=$(CI) pytest \
+		test_${PACKAGE_NAME}/partition/${EXTRA_NAME}
```

Comment:
this isn't so friendly from the cli since EXTRA_NAME won't be defined.

maybe a make target for each EXTRA_NAME ?